frontend/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "petrakip-app",
+  "description": "An Ionic project",
+  "version": "0.0.1",
+  "private": false,
+  "scripts": {
+    "start": "react-scripts start",
+    "serve": "ionic serve",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "lint": "yarn eslint && yarn prettier --check",
+    "fmt": "yarn eslint --fix && yarn prettier --write",
+    "prettier": "prettier \"**/*.{ts,tsx}\"",
+    "eslint": "eslint . --ext .ts"
+  },
+  "dependencies": {
+    "@aws-amplify/ui-react": "^1.0.7",
+    "@capacitor/core": "2.4.7",
+    "@ionic/react": "5.5.0",
+    "@ionic/react-router": "5.5.0",
+    "@testing-library/jest-dom": "5.11.9",
+    "@testing-library/react": "11.2.5",
+    "@testing-library/user-event": "12.6.3",
+    "@types/jest": "26.0.20",
+    "@types/node": "12.19.15",
+    "@types/react": "16.14.3",
+    "@types/react-dom": "16.9.10",
+    "@types/react-router": "5.1.11",
+    "@types/react-router-dom": "5.1.7",
+    "aws-amplify": "^3.3.27",
```

Comment:
probably devDependencies

Reply:
as mentioned ^ everything was created by `ionic-cli` and since we are building it as an app this doesn't matter.